examples/examples-cpu/snowflake/snowflake-write.ipynb

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
this notebook is empty...is that intentional? If you're not adding code for it in this PR, can you please remove it?

Reply:
its a placeholder, but np I'll remove it and the TBD from the readme

Reply:
we should not have unused code or TBD documentation here. This repo is intended to be documentation for Saturn, and having stuff that you have to "just know" to ignore is a distracting, and possibly confusing, experience